Automattic/newspack-content-converter
# Node 12.x Support, Updated NPM Scripts

This PR is a little bit of housekeeping:

- Updates Calypso Build version clearing the way to build with current Node (v.12.14.0). This is based on https://github.com/Automattic/newspack-plugin/pull/355
- Updates npm scripts to mirror Newspack Plugin. Based on https://github.com/Automattic/newspack-plugin/pull/354. The README has been updated to reflect the script changes.

### Testing Instructions

- Switch to current node `e.g. nvm use stable`. `node --version` should now show `12.14.0`
- Execute `npm install`, verify no errors.
- Execute `npm start`. Verify the plugin builds with no errors  and watches for new changes.
- Execute `npm run release:archive` and verify an archive is is built in `assets/release/`
- Verify updates to the README are accurate and clear.
